pkg/katapult/testdata/TestVirtualMachineUpdateArguments_JSONMarshaling/null.golden

```diff
@@ -0,0 +1 @@
+null
```

Comment:
Seems this golden file is left-over from a test we deleted. Easiest way to get rid of all orphaned golden files is to simply run `make regen-golden`, that will delete all existing golden files and re-generate them from scratch.

The new files should all be identical though, with the exception of orphaned ones not being re-created.